src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,45 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const currentTab = tabs.find(({ id }) => id === selectedTabId)
+    || tabs[0];
+  const currentContent = currentTab.content;
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              data-cy="Tab"
+              key={tab.id}
+              className={cn(
+                { 'is-active': currentTab === tab },
+              )}
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => {
+                  if (currentTab !== tab) {
+                    onTabSelected(tab);
+                  }
+                }}
```

Comment:
It's better to create a handler function instead of just passing anonim function to onClick in this case. As it's not a one line of code